app/views/FeatureFlags.js

```diff
@@ -0,0 +1,129 @@
+/* eslint-disable react-native/no-raw-text */
+import React, { useEffect, useState } from 'react';
+import { BackHandler, StyleSheet, View } from 'react-native';
+
+import NavigationBarWrapper from '../components/NavigationBarWrapper';
+import { Switch } from '../components/Switch';
+import { Typography } from '../components/Typography';
+import Colors from '../constants/colors';
+import { DEBUG_MODE } from '../constants/storage';
+import { buildTimeFlags, useFlags } from '../helpers/flags';
+import { GetStoreData } from '../helpers/General';
+import { disableDebugMode, enableDebugMode } from '../helpers/Intersect';
+
+export const FeatureFlagsScreen = ({ navigation }) => {
+  const [flags, setFlag] = useFlags();
+  const [isExposed, setIsExposed] = useState(false);
+
+  useEffect(() => {
+    const handleBackPress = () => {
+      navigation.goBack();
+      return true;
+    };
+
+    BackHandler.addEventListener('hardwareBackPress', handleBackPress);
+
+    (async () => {
+      const isExposed = await GetStoreData(DEBUG_MODE, false);
+      setIsExposed(isExposed);
+    })();
+
+    return () => {
+      BackHandler.removeEventListener('hardwareBackPress', handleBackPress);
+    };
+  }, [navigation]);
+
+  const backToMain = () => {
+    navigation.goBack();
+  };
+
+  const getBuildtimeFlagVal = flagName =>
+    buildTimeFlags[flagName] ? 'On' : 'Off';
+
+  const toggleExposureMode = val => {
+    val ? enableDebugMode() : disableDebugMode();
+    setIsExposed(val);
+  };
+
+  const toggleFlag = flagName => setFlag(flagName, !flags[flagName]);
+
+  const getExposureModeToggle = () => (
+    <View style={styles.toggleRow}>
+      <Switch
+        value={isExposed}
+        onValueChange={toggleExposureMode}
+        trackColor={{ false: Colors.GRAY_BACKGROUND, true: Colors.RED_TEXT }}
+      />
+      <Typography
+        use={'body1'}
+        bold
+        style={
+          isExposed
+            ? { ...styles.toggleRowText, color: Colors.RED_TEXT }
+            : styles.toggleRowText
+        }>
+        Exposure Mode
+      </Typography>
+    </View>
+  );
+
+  const getFlagToggleRow = ([flagName, isFlagActive]) => (
```

Comment:
This is a component, make it a functional component:

`<FlagToggleRow key={} value={} />`

and move it out into the module scope